project-words.txt

```diff
@@ -1,6 +1,7 @@
 abernathyca
 adriaan
 agan
+alexbdebrie
```

Comment:
wonder how did you re-order it?

I find this spell checker really painful to work with, everytime my commits are rejected and I have to add new words 😅 Also finding the position to add the word is time consuming

Reply:
In VS Code, I can use the command palette to "transform to lowercase" and "sort lines ascending". I bet IntelliJ has something similar.

Reply:
Indeed I just found similar options

But the implementation seems different for special chars 😅 


<img width="899" alt="CleanShot 2022-08-08 at 17 00 28@2x" src="https://user-images.githubusercontent.com/749374/183449050-950e9568-a94a-4214-9d0b-f363f64522d8.png">

This is quite annoying to order manually. I suggest we could just ignore the order. If you care about maintaining it why not but I'd rather not block a PR just because of that. Or we should find a way to sort this file automatically as a git hook.

Reply:
Ah yes. I didn't send a PR just because of this. More of a drive-by fix.